### DISCUSSION TITLE
is there any plan for developing ESD coregistration and interferogram calculation

### DISCUSSION THREAD

**Discussion by domino20** [cgi-estonia-space/ALUs#35] [General]
Compared with snap, the ESD coregistration and interferogram calculation is still missing. Any plan for these?

**Comment by kautlenbachs** [comment]
We would like to develop much more than it is possible with this time and budget. Currently we are working on some RAW data focussing tasks. However those aforementioned operations might be possible to implement later this year.

**Reply by domino20** [reply]
This project is quite encouraging. I use this for processing two sentinel-1 datasets, the speed is amazing compared with other insar tools(isce, gmtsar). If the interfeorgram can be generated, i guess many people are willing to use this. Gook luck!

**Reply by kautlenbachs** [reply]
The speed could be even faster up to 4x times of the current one. Currently a lot of GPU idling is happening in the background due to various pipeline architecture decisions.
